### ISSUE TITLE
Collapse do not add aria and collapsed class for parent with element

### ISSUE THREAD

**Issue by Johann-S** [twbs/bootstrap#27203]
Related to this line: https://github.com/twbs/bootstrap/blob/v4-dev/js/src/collapse.js#L297

because `selector` is undefined with a DOM Element or a jQuery element

EDIT:
#27202 should be merged to fix that issue

**Comment by Johann-S** [comment]
Finally I was wrong with #27002 everything is fine 😆
